src/components/AnalysisPage/AnalysisPage.jsx

```diff
@@ -9,13 +9,14 @@ import { Row, Col } from "antd";
 const AnalysisPage = () => {
   const [fileFor3dModel, setFileFor3dModel] = useState(null);
   const [partId, setPartId] = useState(null);
+  const [autoPopAnalysis, setAutoPopAnalysis] = useState(null);
```

Comment:
When add the thickness analysis to this, we can create an object autoPopAnalysis,

`const [autoPopAnalysis, setAutoPopAnalysis] = useState({
    geometry_analysis: null,
    thickness_analysis: null,
  });`

And then I'll make adjustments so everything works alongside each other.